
<html xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head>
    <meta charset="utf-8"/>	
    <title> Clustering summary </title>
    <link rel="stylesheet" href="style1.css">
  </head>

 <h1 > Clustering Summary</h1>
<a href="summary_histogram.png"> <img src="summary_histogram.png" width="700" border="1" > </a><p> <b> Graphical summary of the clustering results. </b> Bars represent superclusters, with their heights and widths corresponding to the numbers of reads in the superclusters (y-axis) and to their proportions in all analyzed reads (x-axis), respectively. Rectangles inside the supercluster bars represent individual clusters. If the filtering of abundant satellites was performed, the affected clusters are shown in green, and their sizes correspond to the adjusted values. Blue and pink background panels show proportions of reads that were clustered and remained single, respectively. Top clusters are on the left of the dotted line. </p><hr><br><br>
 <h2 > Run information:</h2>

<p class='character'>Number of input reads: 10000</p>

<p class='character'>Number of analyzed reads: 10000</p>

<p class='character'>Proportion of reads in top clusters : 14 %</p>

<p class='character'>Cluster merging: No</p>

<p class='character'>Paired-end reads: Yes</p>

 <h2 > Available analyses:</h2>
<p> <a href="tarean_report.html">Tandem repeat analysis</a> </p><p> <a href="cluster_report.html">Cluster annotation</a> </p><p> <a href="supercluster_report.html">Supercluster annotation</a> </p><p> <a href="summarized_annotation.html">Repeat annotation summary</a> </p>
 <h2 > Supplementary files:</h2>
<p> <a href="CLUSTER_TABLE.csv">CLUSTER_TABLE.csv</a> </p><p> <a href="SUPERCLUSTER_TABLE.csv">SUPERCLUSTER_TABLE.csv</a> </p><p> <a href="contigs.fasta">contigs.fasta</a> </p><hr>

<h3> How to cite </h3>
<p>
	Novak, P., Neumann, P., Pech, J., Steinhaisl, J., Macas, J. (2013) -
	  <a href="http://bioinformatics.oxfordjournals.org/content/29/6/792">RepeatExplorer: a Galaxy-based web server for genome-wide characterization of eukaryotic repetitive elements from next generation sequence reads.</a> <i> Bioinformatics</i> <b>29</b>:792-793.
</p>

<p><i> Classification of repetitive elements using REXdb:</i></p>
<p>Neumann, P., Novak, P., Hostakova, N., Macas, J. (2019) &#8211; <a href="https://mobilednajournal.biomedcentral.com/articles/10.1186/s13100-018-0144-1" target="_blank">Systematic survey of plant LTR-retrotransposons elucidates phylogenetic relationships of their polyprotein domains and provides a reference for element classification</a>. <em>Mobile DNA</em> <b>10</b>:1.</p>

</p>
<i>The principle of repeat identification implemented in the RepeatExplorer:</i>
<p>
	  Novak, P., Neumann, P., Macas, J. (2010) - <a href="http://www.biomedcentral.com/1471-2105/11/378">Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data.</a> <i>BMC Bioinformatics</i> <b>11</b>:378.
</p>
<i>Using TAREAN for satellite repeat detection and characterization:</i>
<p>
  Novak, P., Robledillo, L.A.,Koblizkova, A., Vrbova, I., Neumann, P., Macas, J. (2017) -
    <a href="https://doi.org/10.1093/nar/gkx257"> TAREAN: a computational tool for identification and characterization of satellite DNA from unassembled short reads.</a> <i> Nucleic Acid Research </i> <b>45</b>:e111
</p>
<br><hr>
 <h3 > Details:</h3>
<pre>
--------------------------------------------------------------------------
PIPELINE VERSION         : devel-0.3.8-2917(e753f81)

PROTEIN DATABASE VERSION : protein_database_viridiplantae_v3.0.fasta
            md5 checksum : a36362f4e8b024f1ce97589aac1e6f1a

DNA DATABASE VERSION     : dna_database_masked.fasta
            md5 checksum : 86bab7cdd3e70374cd756de13680240d
--------------------------------------------------------------------------
</pre>
<p class='character'>Minimal number of reads in cluster to be considered top cluster : 20</p>

<p class='character'>Reserved Memory : 31G</p>

<p class='character'>Maximum number of processable reads with the reserved memory : 1557523</p>
